pkg/benchmark/resources.go

```diff
@@ -0,0 +1,176 @@
+	cmd := exec.Command("oc", "debug", "node/"+nodeName, "--", "chroot", "/host", "lsblk", "-d", "-n", "-o", "NAME")
+	output, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get disks for node %s: %v", nodeName, err)
+	}
+	disks := parseLines(output)
+
+	mountedDisks, err := getMountedDisks(nodeName)
+	if err != nil {
+		return nil, err
+	}
+
+	return filterUnmountedDisks(disks, mountedDisks), nil
+}
+
+func getMountedDisks(nodeName string) ([]string, error) {
+	cmd := exec.Command("oc", "debug", "node/"+nodeName, "--", "chroot", "/host", "lsblk", "-o", "NAME,MOUNTPOINT")
+	output, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get mounted disks for node %s: %v", nodeName, err)
+	}
+	return parseMountedDisks(output), nil
+}
+
+func parseMountedDisks(output []byte) []string {
+	lines := strings.Split(string(output), "\n")
+	var mountedDisks []string
+	for _, line := range lines {
+		fields := strings.Fields(line)
+		if len(fields) > 1 && fields[1] != "" { // Disk has a mountpoint
+			mountedDisks = append(mountedDisks, fields[0])
+		}
+	}
+	return mountedDisks
+}
+
+func filterUnmountedDisks(allDisks, mountedDisks []string) []string {
+	mountedSet := make(map[string]bool)
+	for _, disk := range mountedDisks {
+		mountedSet[disk] = true
+	}
+
+	var unmountedDisks []string
+	for _, disk := range allDisks {
+		if !mountedSet[disk] {
+			unmountedDisks = append(unmountedDisks, disk)
+		}
+	}
+	return unmountedDisks
+}
+
+func parseLines(output []byte) []string {
+	lines := strings.Split(string(output), "\n")
+	var result []string
+	for _, line := range lines {
+		trimmed := strings.TrimSpace(line)
+		if trimmed != "" {
+			result = append(result, trimmed)
+		}
+	}
+	return result
+}
+
+func getNodeNics(nodeName string) ([]string, error) {
+	cmd := exec.Command("oc", "debug", "node/"+nodeName, "--", "chroot", "/host", "ip", "-o", "link", "show")
+	output, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get NICs for node %s: %v", nodeName, err)
+	}
+	return filterPhysicalInterfaces(parseNicLines(output)), nil
+}
+
+func parseNicLines(output []byte) []string {
+	lines := strings.Split(string(output), "\n")
+	var interfaces []string
+	for _, line := range lines {
+		fields := strings.Fields(line)
+		if len(fields) > 1 {
+			iface := strings.TrimSuffix(fields[1], ":") // Ensure we remove only the colon
+			interfaces = append(interfaces, iface)
+		}
+	}
+	return interfaces
+}
+
+func filterPhysicalInterfaces(nics []string) []string {
+	var filtered []string
+	for _, nic := range nics {
+		if strings.HasPrefix(nic, "lo") || // Ignore loopback
+			strings.HasPrefix(nic, "br-") || // Ignore OVS bridges
+			strings.HasPrefix(nic, "ovs-") || // Ignore Open vSwitch
+			strings.HasPrefix(nic, "ovn-") || // Ignore OVN tunnels
+			strings.HasPrefix(nic, "genev_sys") || // Ignore geneve tunnels
+			len(nic) > 15 { // Ignore hashed virtual interfaces
```

Comment:
```suggestion
		if strings.HasPrefix(nic, "lo") || // Ignore loopback
			strings.HasPrefix(nic, "br-") || // Ignore OVS bridges
			strings.HasPrefix(nic, "ovs-") || // Ignore Open vSwitch
			strings.HasPrefix(nic, "ovn-") || // Ignore OVN tunnels
			strings.HasPrefix(nic, "genev_sys") || // Ignore geneve tunnels
			len(nic) > 15 { // Ignore hashed virtual interfaces
```

Any reason for ignoring those? could be helpful in case of multus cc @BlaineEXE

Reply:
Virtual interfaces have lot of jitter, using them for benchmarking is not useful, there is no meaningful data that we can get.